content/versions-comparison/Serenity-news.json

```diff
@@ -112,49 +112,49 @@
     "link": "/pim/serenity/updates/2020-03.html#new-measurement-units-and-families"
   },
   {
-    "title":"Automatized attributes concatenation 🤖",
+    "title":"Concatenate attributes with the rules engine 🤖",
     "domains": ["Rules engine", "Productivity"],
     "editions": ["CE","EE"],
     "since-version": "Serenity",
     "link": "/pim/serenity/updates/2020-03.html#a-new-rule-action-to-concatenate-attributes"
   },
   {
-    "title":"Product export monitoring 📊",
+    "title":"Monitor your destination connections 📊",
     "domains": ["Connectivity","Reporting"],
     "editions": ["CE","EE"],
     "since-version": "Serenity",
     "link": "/pim/serenity/updates/2020-04.html#track-your-destination-connections"
   },
   {
-    "title":"Connection errors easy fixing",
+    "title":"Improved error management on connections",
     "domains": ["Connectivity","Reporting"],
     "editions": ["CE","EE"],
     "since-version": "Serenity",
     "link": "/pim/serenity/updates/2020-07.html#easily-fix-errors-on-your-connections"
   },
   {
-    "title":"Automatized attribute computation 🤖",
+    "title":"Calculate attribute values with the rules engine 🤖",
     "domains": ["Rules engine","Productivity"],
     "editions": ["CE","EE"],
     "since-version": "Serenity",
     "link": "/pim/serenity/updates/2020-04.html#calculate-attribute-values"
   },
   {
-    "title":"Error monitoring on product import 📊",
+    "title":"Improved error monitoring on connections 📊",
     "domains": ["Connectivity","Reporting"],
     "editions": ["CE","EE"],
     "since-version": "Serenity",
     "link": "/pim/serenity/updates/2020-05.html#easily-monitor-errors-on-your-connections"
   },
   {
-    "title":"Spell-check on attribute options",
+    "title":"Spell-check on attribute options and labels",
     "domains": ["Quality"],
     "editions": ["CE","EE"],
     "since-version": "Serenity",
     "link": "/pim/serenity/updates/2020-07.html#avoid-spelling-mistakes-in-your-product-data"
   },
   {
-    "title":"Attribute options easy management",
+    "title":"Attribute options screen revamp",
```

Comment:
Screen revamp doesn't help to understand why we did it. It's not the feature, "the revamp".